networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,67 @@
+"""
+********
+Covering
+********
+"""
+
+#    Copyright (C) 2015 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+import networkx as nx
+from networkx.utils import not_implemented_for, arbitrary_element
+
+
+@not_implemented_for('directed', 'multigraph')
+def min_edge_cover(G):
+    """Returns a set of edges which constitute the minimum edge cover of the graph.
+    An edge cover of a graph is a set of edges such that every vertex of
```

Comment:
The first line of the docstring should have a blank line after it to indicate that it is a single-sentence summary.